src/insights_client/run.py

```diff
@@ -46,8 +50,8 @@
         event["ended_at"] = utc.make_utc_datetime_rfc3339()
         try:
             metrics_client.post(event)
-        except OSError as e:
-            print("Error: Could not submit event: {0}".format(e))
+        except (OSError, IOError, ConnectionError) as e:
```

Comment:
ConnectionError [is](https://github.com/psf/requests/blob/1466ad713cf84738cd28f1224a7ab4a19e50e361/requests/exceptions.py#L12) IOError.

```suggestion
        except (OSError, IOError) as e:
```

should be enough and it would save us from importing requests here.